src/main/resources/META-INF/frontend/fcEnhancedTabs/fc-enhanced-tabs.css

```
@@ -98,3 +98,39 @@ vaadin-menu-bar-button[theme~='fc-enhanced-tabs'] vaadin-menu-bar-item {
   --aura-surface-opacity: 0.3;
 }
+
+vaadin-menu-bar-button[theme~='fc-enhanced-tabs'] {
+  --focus-ring-width: var(--vaadin-focus-ring-width, 2px);
+}
+
+vaadin-menu-bar-button[theme~='fc-enhanced-tabs'] vaadin-tab {
+  --_lumo-flag: var(--vaadin-lumo-theme, 0);
+
+  padding-block: calc(
+    (var(--_lumo-flag) * 0.5rem) + 
```

Stylelint (17.3.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)